content/sensu-go/5.13/installation/install-sensu.md

```diff
@@ -254,13 +254,13 @@ sudo yum install sensu-go-agent
 
 {{< highlight "Windows" >}}
 # Download the Sensu agent for Windows amd64
-Invoke-WebRequest https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.13.0/sensu-go-agent_5.13.0.5887_en-US.x64.msi  -OutFile "$env:userprofile\sensu-go-agent_5.13.0.5887_en-US.x64.msi"
+Invoke-WebRequest https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.13.1/sensu-go-agent_5.13.1.5887_en-US.x64.msi  -OutFile "$env:userprofile\sensu-go-agent_5.13.1.5887_en-US.x64.msi"
```

Comment:
Unfortunately we include a build number in the filename, which is also going to change. That is, it won't be `5887` in the filename anymore, but another number I'll get later in the process.

I see no reason to include a build number in the file name; we already reference a release's build number in our release checklist. I'll create a retro card to get rid of that number in the filename.

Reply:
thanks @ccressent, this will almost make updating the downloads page easier as well.

Reply:
Totally; that will mean you only need to know the release version number to update all those links, which can then be done in advance if one so wishes. No need to wait for eng to tell you the build number.

For this patch release though, the build number is `5957`.

Reply:
@ccressent to confirm it'll be `https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.13.1/sensu-go-agent_5.13.1_en-US.x64.msi`   then?

Reply:
The new build number, `5957`, will still be in the file name for the files ending in `.msi`, `.rpm` and `.deb`. For example: `https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.13.1/sensu-go-agent_5.13.1.5957_en-US.x64.msi`

See the files on S3: https://s3.console.aws.amazon.com/s3/buckets/sensu.io/sensu-go/5.13.0/?region=us-west-2&tab=overview